ee-9/feature-pack/src/main/resources/modules/system/layers/base/com/sun/xml/bind/main/module.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<module xmlns="urn:jboss:module:1.9" name="com.sun.xml.bind">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${com.sun.istack:istack-commons-runtime}"/>
+        <artifact name="${com.sun.istack:istack-commons-tools}"/>
+        <artifact name="${com.sun.xml.bind.external:rngom}"/>
+        <artifact name="${com.sun.xml.bind.external:relaxng-datatype}"/>
+        <artifact name="${org.glassfish.jaxb:codemodel}"/>
+        <artifact name="${org.glassfish.jaxb:jaxb-core}"/>
+        <artifact name="${org.glassfish.jaxb:jaxb-jxc}"/>
+        <artifact name="${org.glassfish.jaxb:jaxb-runtime}"/>
+        <artifact name="${org.glassfish.jaxb:jaxb-xjc}"/>
+        <artifact name="${org.glassfish.jaxb:xsom}"/>
+        <artifact name="${org.glassfish.jaxb:txw2}"/>
+    </resources>
+
+    <dependencies>
+        <module name="java.logging"/>
+        <module name="java.xml"/>
+        <module name="javax.activation.api"/>
+        <module name="javax.api"/>
```

Comment:
The other modules are already fine grained. Is this depracted one really necessary? Shall I help to check the jars?